.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Ticket
+🎫 [TICKET_ID] <!-- Ticket ID, if there's any put it between brackets -->
+
+### Description
+<!-- Please describe what you have changed or added -->
+<!-- What types of changes does your code introduce? -->
+<!-- Bug fix (non-breaking change which fixes an issue) -->
+<!-- New feature (non-breaking change which adds functionality) -->
+<!-- Include any important information for reviewers -->
+<!-- Etc, etc, etc -->
```

Comment:
Maybe we should create a "good" PR and link to it here so people have an example?

Reply:
That's a neat idea, maybe we can add this once we have that PR?